### ISSUE TITLE
请问如何设计正则化来防止focal loss带来的过拟合会比较好？可以帮忙给出一个样例参考吗？

### ISSUE THREAD

**Issue by qianhuiliu** [yatengLG/Focal-Loss-Pytorch#20]
你好，非常感谢你的代码贡献。我在现在这样简单的优化设定下过拟合很严重：
optimizer = torch.optim.SGD(model.parameters(), lr=lr, momentum=0.9, weight_decay=1e-5)
scheduler = StepLR(optimizer, step_size=40, gamma=0.1)
看到你之前的回复说需要设置正则化来抗过拟合，请问可以帮忙给出一个正则化的方式吗？非常感谢！
